mirrord/layer/src/load.rs

```diff
@@ -0,0 +1,77 @@
+use mirrord_config::{util::VecOrSingle, LayerConfig};
+
+#[cfg(target_os = "macos")]
+mod sip {
+    use std::{collections::HashSet, sync::LazyLock};
+
+    static SIP_EXEC_PROCESSES: LazyLock<HashSet<&str>> =
+        LazyLock::new(|| HashSet::from(["sh", "bash", "cargo", "rustc", "cargo-watch"]));
```

Comment:
I don't agree on the sh/bash - it might be beneficial for users to load into those.

Reply:
it's only if specified in `--skip-processes sh`

Reply:
ah, okay need better docs 🖌️